### PR TITLE
Fix Giant Fly Wing bugs

### PR DESCRIPTION
**Pull request by kaninhot004**
* **Addressed Issue(s)**: -

* **Server Mode**: Renewal

* **Description of Pull Request**: Fix Giant Fly Wing bugs, the normal one (Not unlimited) not consume when used.

This is how it look when bugged.

https://github.com/rathena/rathena/assets/9493525/7ac5c2db-38c7-431b-af60-a529896ec2bb

### CODE REVIEW

File: db/re/item_db_usable.yml [hunk 1]
```diff
@@ -47035,7 +47035,7 @@ Body:
       NoMail: true
       NoAuction: true
     Script: |
-      rentitem 12212,432000;
+      rentitem 23023,432000;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:47038]: This box should give a rental fly wing for 5d (id 12887 I guess) according to kRO description.
Also our english item name is currently wrong.

Comment (kaninhot004) [reply db/re/item_db_usable.yml:47038]: Thanks I will check and fix later.

**Review by Atemo — APPROVED** [review]

**Comment by aleos89** [comment]
It looks like these were changed in #5859 to support the script function. I can't remember if we changed functionality of Useable and DelayConsume in the past 2 years now. Just leaving this note before it's merged.

**Comment by Daegaladh** [comment]
> It looks like these were changed in #5859 to support the script function. I can't remember if we changed functionality of Useable and DelayConsume in the past 2 years now. Just leaving this note before it's merged.

Maybe related to my PR too https://github.com/rathena/rathena/pull/6204

**Comment by aleos89** [comment]
So it seems like the solution is to add that sleep back in to the function.

**Comment by Daegaladh** [comment]
> So it seems like the solution is to add that sleep back in to the function.

Not at all, the sleep was necessary because the itemskill+warpparty, but now it's an if/else, so changing it back to Usable would work, or we can just move the entire function to the item script.

**Comment by aleos89** [comment]
Gotcha, makes sense!

**Review by aleos89 — APPROVED** [review]

**Pull request merged**